index.html

```diff
@@ -31,7 +31,167 @@ <h1>Charlie Giorgania</h1>
       </div>
 
-      <div></div>
+      <section class="projectCards">
+        <div>
+          <h2 class="projects">Projects</h2>
+        </div>
+
+        <div class="projImg"></div>
+
+        <div class="container">
+          <p class="projName">Project name goes here</p>
+          <ul class="stacks">
+            <li>HTML/CSS</li>
+            <li>Ruby on Rails</li>
+            <li>JavaScript</li>
+          </ul>
+
+          <button class="projView">
+            See this Project
+            <img class="arrow" src="./images/arrow.png" alt="" />
+          </button>
+        </div>
+
+        <div class="projImg2"></div>
+
+        <div class="container2">
+          <p class="projName">Project name goes here</p>
+          <ul class="stacks">
+            <li>HTML/CSS</li>
+            <li>Ruby on Rails</li>
+            <li>JavaScript</li>
+          </ul>
+
+          <button class="projView">
+            See this Project
+            <img class="arrow2" src="./images/arrow.png" alt="" />
+          </button>
+        </div>
+
+        <div class="projImg3"></div>
+
+        <div class="container3">
+          <p class="projName">Project name goes here</p>
+          <ul class="stacks">
+            <li>HTML/CSS</li>
+            <li>Ruby on Rails</li>
+            <li>JavaScript</li>
+          </ul>
+
+          <button class="projView">
+            See this Project
+            <img class="arrow" src="./images/arrow.png" alt="" />
+          </button>
+        </div>
+
+        <div class="projImg4"></div>
+
+        <div class="container4">
+          <p class="projName">Project name goes here</p>
+          <ul class="stacks">
+            <li>HTML/CSS</li>
+            <li>Ruby on Rails</li>
+            <li>JavaScript</li>
+          </ul>
+
+          <button class="projView">
+            See this Project
+            <img class="arrow" src="./images/arrow.png" alt="" />
+          </button>
+        </div>
+
+        <div class="projImg5"></div>
+
+        <div class="container5">
+          <p class="projName">Project name goes here</p>
+          <ul class="stacks">
+            <li>HTML/CSS</li>
+            <li>Ruby on Rails</li>
+            <li>JavaScript</li>
+          </ul>
+
+          <button class="projView">
+            See this Project
+            <img class="arrow" src="./images/arrow.png" alt="" />
+          </button>
+        </div>
+
+        <div class="projImg6"></div>
+
+        <div class="container6">
+          <p class="projName">Project name goes here</p>
+          <ul class="stacks">
+            <li>HTML/CSS</li>
+            <li>Ruby on Rails</li>
+            <li>JavaScript</li>
+          </ul>
+
+          <button class="projView">
+            See this Project
+            <img class="arrow" src="./images/arrow.png" alt="" />
+          </button>
+        </div>
+      </section>
+
+      <section>
+        <div class="aboutMe"><h2>About</h2></div>
```

Comment:
[OPTIONAL]
- [ ] I suggest you have the content aligned with flexbox. This will make your app more responsive.